dbms/src/Flash/Coprocessor/DAGContext.cpp

```diff
@@ -75,6 +75,11 @@ std::unordered_map<String, BlockInputStreams> & DAGContext::getProfileStreamsMap
     return profile_streams_map;
 }
 
+void DAGContext::updateFinalConcurrency(size_t cur_streams_size, size_t max_streams)
```

Comment:
May rename max_streams param to names like streams_upper_limit? Since max always go with std::max().

Reply:
ok, renamed